src/diffusers/pipelines/aura_flow/pipeline_aura_flow.py

```diff
@@ -111,7 +112,7 @@ def retrieve_timesteps(
     return timesteps, num_inference_steps
 
 
-class AuraFlowPipeline(DiffusionPipeline):
+class AuraFlowPipeline(DiffusionPipeline, FromSingleFileMixin):
```

Comment:
```suggestion
class AuraFlowPipeline(DiffusionPipeline):
```
Single file pipeline support e.g. [aura_flow_0.3.safetensors](https://huggingface.co/fal/AuraFlow-v0.3/blob/main/aura_flow_0.3.safetensors) requires further changes, no?